src/components/ComposerQuill/composer.js

```diff
@@ -206,7 +207,14 @@ class Composer extends React.Component {
       const enableMarkdown = !markdown?.disabled;
 
       // gets the text from the composer with mentions as a placeholder string
-      const text = getQuillText(this.quill);
+      let text = getQuillText(this.quill);
+
+      const results = String.raw`${text}`.match(/\\/g);
+
+      if (results) {
+        // replace backslash and underscore with htmlcode
+        text = String.raw`${text}`.replace(/\\/g, '&bsol;').replace(/_/g, '&#95;');
```

Comment:
The reason why the shrug doesn't work is because of markdown. The backslash is used to escape the markdown characters underscore and asterisk. So when we do the shrug emoticon `¯\_(ツ)_/¯` the backslash escapes the underscore.

After thinking about it, I'm not sure we can make this change. This would mean we wouldn't ever be able to escape the underscore for legit reasons because it would always add the escape character. Reddit also has this problem because they use markdowns. Teams desktop also has this problem because of markdown.